staff/alejandro-perez/arrays-native/scripts/concat.test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+describe('Array.prototype.concat', function () {
+    var array;
+    var array1;
+
+    beforeEach(function () {
+        array = ['a','b','c']
```

Comment:
You missed semicolon! :(

Reply:
He thinks that works in EcmaScript 6....